rocky/rocky/templates/partials/form/field_input.html

```diff
@@ -1,7 +1,8 @@
 {% load i18n %}
 
 <div {% if field.field.required %}class="required"{% endif %}>
-    {{ field.label_tag }}
+    <p class="de-emphasized">{{ field.label_tag }}</p>
+    <p class="nota-bene">{{ field.field.widget.attrs.description }}</p>
```

Comment:
How does the `description` differ from the `help_text`? If the `description` "belongs to" the input field, the input should probably refer to it by `id` in its `aria-describedby`.

Reply:
> How does the `description` differ from the `help_text`?

The `help_text` creates an info button next to the input, which expands when clicked. For this design, some input fields contain only a short description, which should be easily visible to every user. Therefore, a `nota-bene` is requested for the smaller descriptions. 

I've now added an `aria-describedby` to the input fields which refers to the nota-bene description.